src/poprox_recommender/default.py

```diff
@@ -173,6 +173,53 @@ def mmr_diversification(rewards, similarity_matrix, theta, topk):
             S.append(candidate)
     return S  # LIST(candidate index)
 
+def pfar_diversification(rewards, articles, lamb, tau, topk):
```

Comment:
To write an experiment manifest that uses both kinds of diversification we have now, we'll need to be able to provide a different URL for each one. We'll either want to set up two separate endpoints (one for MMR and one for PFAR) or a parameter on the request that switches between them.